CONDUCT.md

```diff
@@ -65,18 +68,46 @@ If someone reports harassment, ask for a written account of what happened. This
 
 If the following information is not volunteered in a report, ask for it/include it, but do not pressure anyone:
 
-- Identifying information (name/badge number) of the participant doing the harassing
-- The behavior that was in violation
-- The approximate time of the behavior (if different than the time the report was made)
-- The circumstances surrounding the incident
-- Other people involved in the incident
-
-Further resources:
-
-- [geekfeminism.wikia.com/wiki/Conference_anti-harassment/Responding_to_reports](http://geekfeminism.wikia.com/wiki/Conference_anti-harassment/Responding_to_reports)
-- [wiki.openhatch.org/OpenHatch_event_code_of_conduct/Staff_Procedures_for_Handling_Incidents](http://wiki.openhatch.org/OpenHatch_event_code_of_conduct/Staff_Procedures_for_Handling_Incidents)
+* identifying information (name/badge number) of the participant doing the harassing
+* the behavior that was in violation
+* the approximate time of the behavior (if different than the time the report was made)
+* the circumstances surrounding the incident
+* other people involved in the incident
 
 #### Email Reporting
 
 Our `coc@ournetworks.ca` is monitored by an individual on a rotating basis, new monitors will be determined at a planning meeting once per year.
 
+#### Handling Reports
+
+The following will be in place at the event to handle reports: 
+
+<<<<<<< HEAD
```

Comment:
@dcwalk This and lines `107`, `113`,  looks like lines from the merge conflict?

Reply:
yes, thank you... thought I fixed locally